docs/source/tutorial/typesfuns.rst

```diff
@@ -263,7 +261,7 @@ You can override this with a ``partial`` annotation:
 
 However, this is not advisable, and in general you should only do this during
 the initial development of a function, or during debugging.  If you try to
-evaluate ``fromMaybe Nothing`` at run time you will get a run time error.
+evaluate ``fromMaybe Nothing`` at runtime, you will get a runtime error.
```

Comment:
I believe this is subtly different: I've seen "run time" (or "run-time") used to distinguish from "compile time", with "runtime" typically referring to the underlying language/system which implements the I/O etc.

Reply:
As there is no 'mainstream convention', the key thing here is that we are internally consistent within our documentation.